docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -799,22 +837,28 @@ public class BearerTokenAuthorizationTest {
             .when().get("/api/users/preferredUserName")
             .then()
             .statusCode(200)
-            // the test endpoint returns the name extracted from the injected SecurityIdentity Principal
+            // The test endpoint returns the name extracted from the injected SecurityIdentity Principal
             .body("userName", equalTo("alice"));
     }
 
 }
 ----
 
-`KeycloakTestResourceLifecycleManager` registers `alice` and `admin` users. The user `alice` has the `user` role only by default - it can be customized with a `keycloak.token.user-roles` system property. The user `admin` has the `user` and `admin` roles by default - it can be customized with a `keycloak.token.admin-roles` system property.
+.Summary
+
+In the provided example, `KeycloakTestResourceLifecycleManager` registers two users: `alice` and `admin`.
+By default:
+* The user `alice` has the `user` role, which you can customize by using a `keycloak.token.user-roles` system property.
+* The user `admin` has both the `user` and `admin` roles, which you can customize by using the `keycloak.token.admin-roles` system property.
 
-By default, `KeycloakTestResourceLifecycleManager` uses HTTPS to initialize a Keycloak instance which can be disabled with `keycloak.use.https=false`.
-Default realm name is `quarkus` and client id - `quarkus-service-app` - set `keycloak.realm` and `keycloak.service.client` system properties to customize the values if needed.
+By default, `KeycloakTestResourceLifecycleManager` uses HTTPS to initialize a Keycloak instance, and this can be disabled by using `keycloak.use.https=false`.
+The default realm name is `quarkus`, and the client id is `quarkus-service-app`.
+If you want to customize these values, set the `keycloak.realm` and `keycloak.service.client` system properties.
 
 [[integration-testing-public-key]]
-==== Local Public Key
+==== Using a local public key
 
-You can also use a local inlined public key for testing your `quarkus-oidc` `service` applications:
+You can use a local inline public key for testing your `quarkus-oidc` `service` applications, as shown in the following example:
```

Comment:
`inline` is not correct in this context I believe

Reply:
@sberyozkin: Can you help clarify what is meant by "inlined" (versus "inline"). Vale is throwing a warning for "inlined" and it is not in the Merriam-Webster dictionary either.  From my understanding of the context, would it be correct to say "You can use a local public key inline for testing your.........applications,....."  (just move the placement of "inline" in the sentence?